cg/server/api.py

```diff
@@ -356,6 +366,23 @@ def parse_flow_cell(flowcell_id):
     return jsonify(**flow_cell.to_dict(samples=True))
 
 
+@BLUEPRINT.route("/flowcells/<flow_cell_id>/sequencing_metrics", methods=["GET"])
+def get_sequencing_metrics(flow_cell_id: str):
```

Comment:
```suggestion
def get_sequencing_metrics(flow_cell_name: str):
```
Worth considering that we retrieve using the flow cell name and not the database entry id. Can also be read from the function usage below I suppose, more a comment on consistency. Although the end-point is coded to retrieve for an id. So I get the choice here as well.  I suppose here the endpoint might clash with the back-end.

Reply:
I think we should go with flow_cell_name, since that is the column title. However `flow_cell_id` is used a lot in cg 😬

Reply:
Yeah, I'll change it to `flow_cell_name`, it bothers me a little since name does not imply uniqueness and resources are usually fetched with an id. But it is more consistent and something we can change in a separate PR

Reply:
https://github.com/Clinical-Genomics/cg/pull/2268/commits/afa6d3f0318a5d5dc830321f6b0ddf112c6c9dfe